app/fastlane/helpers/ios.rb

```
@@ -0,0 +1,150 @@
+        api_key = ios_connect_api_key
+
+        latest = Fastlane::Actions::LatestTestflightBuildNumberAction.run(
+          api_key: api_key,
+          app_identifier: ENV["IOS_APP_IDENTIFIER"],
+          platform: "ios",
+        )
+
+        new_number = latest + 1
+        Fastlane::Actions::IncrementBuildNumberAction.run(
+          build_number: new_number,
+          xcodeproj: xcodeproj,
```

RuboCop (1.76.1)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)